spec/index.bs

```diff
@@ -1251,7 +1251,7 @@ To get a {{GPUDevice}}, use {{GPUAdapter/requestDevice()}}.
 interface GPUDevice : EventTarget {
     [SameObject] readonly attribute GPUAdapter adapter;
     readonly attribute FrozenArray<GPUExtensionName> extensions;
-    readonly attribute object limits;
+    readonly attribute GPULimits limits;
```

Comment:
Attributes can't be dictionaries: https://heycam.github.io/webidl/#idl-attributes
(I knew there was a reason this was this way, but took me a while to remember.)

Reply:
Ah, yeah! So really the right pattern would probably be to make `GPULimits` an interface if you intend to expose it like this, and have a separate dictionary (`GPULimitsDescriptor`?) that gets passed in by the developer. [We did this in WebXR in a few places](https://immersive-web.github.io/webxr/#xrrenderstate-interface). It's annoying to spec but there are a few perks to it.

Regardless, I'll take that change out of this PR.

Reply:
We talked about doing that, but interfaces are rather heavyweight and we don't need a prototype chain (or to pollute the global scope) for this. And we don't need the implementations of the attributes to go out to C++ (they can just be JS values). So not sure what the right thing is; I don't think we've even tried to implement it.